tracer/src/Datadog.Trace/Telemetry/TelemetrySettings.cs

```diff
@@ -8,12 +8,15 @@
 using System;
 using Datadog.Trace.Configuration;
 using Datadog.Trace.Configuration.Telemetry;
+using Datadog.Trace.Logging;
 using Datadog.Trace.Util;
 
 namespace Datadog.Trace.Telemetry
 {
     internal class TelemetrySettings
     {
+        private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(TelemetrySettings));
```

Comment:
No, and adding it makes me a little nervous given that these settings are also used to _configure_ logging IIRC 😅 

```suggestion
```

Reply:
removed in 63462c108